CenterForOpenScience/ember-osf-preprints
# [IN-22][Preprints] Fix author list with long names and small screens

## Purpose
Fix Author list with long names and smaller screens. The problem shows with Long name strings or small window sizing.

## Summary of Changes

- Update styling for author table (use ellipses and tooltip) 


## Side Effects / Testing Notes
1. Confirm various sizes render the name or cut it off if too long.
2. Confirm that no buttons are lost with names that are too long, short or various window widths (this occurred while I was right-sizing the max-width)

After Alex's modification, slightly more text visible for name col:
<img width="568" alt="screen shot 2018-04-04 at 5 29 25 pm" src="https://user-images.githubusercontent.com/1322421/38336071-8d57da0e-382e-11e8-8abc-89bb8ba8ba4e.png">
<img width="731" alt="screen shot 2018-04-04 at 5 29 33 pm" src="https://user-images.githubusercontent.com/1322421/38336078-91a2abc0-382e-11e8-9423-d83c5e2911ff.png">



## Ticket

https://openscience.atlassian.net/browse/IN-22

# Reviewer Checklist
